src/features/events/store.ts

```diff
@@ -629,22 +622,14 @@ function addEventToState(state: EventsStoreSlice, events: ZetkinEvent[]) {
 
     const campaign = event.campaign;
     if (campaign) {
-      if (!state.eventsByCampaignId[campaign.id]) {
-        state.eventsByCampaignId[campaign.id] = remoteList();
-      }
+      const remoteEventsById = getOrCreateList(
+        state.eventsByCampaignId,
+        campaign.id.toString()
```

Comment:
Had to typecast this to string. I'm assuming it was done this implicitly before. It's because the eventsByCampaignId record is of type <String, RemoteList<...>>, so the key is string. But the ID is a number, hence the typecast.